apps/mull-ui/src/environments/environment.ts

```diff
@@ -4,4 +4,5 @@
 export const environment = {
   production: false,
   backendUrl: 'http://localhost:3333',
+  backendWsUrl: 'ws://localhost:3333',
```

Comment:
Pleas don't forget to add prod equivalent

Reply:
Where should that be since I can't seem to find one for the backend Url in the frontend app?

Reply:
It seems that with our deployment, WebSockets are handled by NGINX out of the box. So we can leave the prod equivalent with an empty string for now.

Ref: https://kubernetes.github.io/ingress-nginx/user-guide/miscellaneous/#websockets